packages/manager/apps/hub/src/shell.js

```diff
@@ -0,0 +1,19 @@
+let shellClient;
+
+export const setShellClient = (client) => {
+  shellClient = client;
+
+  // set callbacks
+  // set on locale change
+  shellClient.i18n.onLocaleChange(() => {
+    window.location.reload();
```

Comment:
Don't know if we need to reload `window.top`?

Reply:
do we need to reload the app-shell to see changes ? if so we need to reload window.top too, if it's dynamically updated in the react app-shell then it's ok

Reply:
For now, I think we need to reload top window as in the app-shell, we don't manage i18n refresh.